gradle/libs.versions.toml

```diff
@@ -11,7 +11,7 @@ core-ktx = "1.8.0"
 detekt = "1.23.1"
 
 ## E
-espresso = "3.5.1"
+espresso = "3.4.0"
```

Comment:
Do we have any reason to downgrade the version of Espresso?

Reply:
I added `journey-test` and the last version of this journey uses espresso 3.4.0, so there is a conflict in the versions